src/applications/vaos/new-appointment/components/DateTimeSelectPage/index.jsx

```diff
@@ -77,178 +80,186 @@ function ErrorMessage({ facilityId, startRequestFlow }) {
+    },
+    [getAppointmentSlots, preferredDate],
+  );
 
-  render() {
-    const {
-      appointmentSlotsStatus,
-      availableDates,
-      availableSlots,
-      data,
-      eligibleForRequests,
-      facilityId,
-      pageChangeInProgress,
-      preferredDate,
-      timezone,
-      timezoneDescription,
-      typeOfCareId,
-    } = this.props;
+  useEffect(
+    () => {
+      if (validationError && submitted) {
+        scrollAndFocus('.usa-input-error-message');
+      }
+    },
+    [validationError, submitted],
+  );
 
-    const calendarData = data?.calendarData || {};
-    const { currentlySelectedDate, selectedDates } = calendarData;
-    const startMonth = preferredDate
-      ? moment(preferredDate).format('YYYY-MM')
-      : null;
+  const calendarData = data?.calendarData || {};
+  const { currentlySelectedDate, selectedDates } = calendarData;
+  const startMonth = preferredDate
+    ? moment(preferredDate).format('YYYY-MM')
+    : null;
 
-    return (
-      <div>
-        <h1 className="vads-u-font-size--h2">{pageTitle}</h1>
-        {appointmentSlotsStatus !== FETCH_STATUS.loading && (
-          <WaitTimeAlert
-            eligibleForRequests={eligibleForRequests}
+  return (
+    <div>
+      <h1 className="vads-u-font-size--h2">{pageTitle}</h1>
+      {appointmentSlotsStatus !== FETCH_STATUS.loading && (
+        <WaitTimeAlert
+          eligibleForRequests={eligibleForRequests}
+          facilityId={facilityId}
+          nextAvailableApptDate={availableSlots?.[0]?.start}
+          onClickRequest={startRequestAppointmentFlow}
+          preferredDate={preferredDate}
+          timezone={timezoneDescription}
+          typeOfCareId={typeOfCareId}
+        />
+      )}
+      {appointmentSlotsStatus !== FETCH_STATUS.failed && (
+        <p>
+          Please select a desired date and time for your appointment.
+          {timezone &&
+            ` Appointment times are displayed in ${timezoneDescription}.`}
+        </p>
+      )}
+      <CalendarWidget
+        monthsToShowAtOnce={2}
+        maxSelections={1}
+        availableDates={availableDates}
+        currentlySelectedDate={currentlySelectedDate}
+        selectedDates={selectedDates}
+        additionalOptions={{
+          fieldName: 'datetime',
+          required: true,
+          maxSelections: 1,
+          getOptionsByDate: selectedDate =>
+            getOptionsByDate(selectedDate, timezone, availableSlots),
+        }}
+        loadingStatus={appointmentSlotsStatus}
+        loadingErrorMessage={
+          <ErrorMessage
             facilityId={facilityId}
-            nextAvailableApptDate={availableSlots?.[0]?.start}
-            onClickRequest={this.props.startRequestAppointmentFlow}
-            preferredDate={preferredDate}
-            timezone={timezoneDescription}
-            typeOfCareId={typeOfCareId}
+            startRequestFlow={() =>
+              startRequestFlow({ requestAppointmentDateChoice, history })
+            }
```

Comment:
You should be able to replace these lines with:

`startRequestFlow={() => requestAppointmentDateChoice(history)}`